src/platforms/esp32/components/avm_builtins/gpio_driver.c

```diff
@@ -63,7 +63,7 @@ static void gpio_driver_init(GlobalContext *global);
 
 #ifdef CONFIG_AVM_ENABLE_GPIO_PORT_DRIVER
 static NativeHandlerResult consume_gpio_mailbox(Context *ctx);
-static void IRAM_ATTR gpio_isr_handler(void *arg);
+static void gpio_isr_handler(void *arg);
```

Comment:
Which warning are we fixing here?

Reply:
The attribute is there twice (for prototype and implementation), and it should be there only once.
Otherwise, second occurrence is ignored with a warning. I checked other uses in esp idf, they only put it for implementation.

See latest branch build with esp idf 5.2:

https://github.com/atomvm/AtomVM/actions/runs/9051457733/job/24867998279#step:4:1271

Reply:
Thanks, understood.